csharp/src/Microsoft.AI.MachineLearning.Interop/Microsoft.AI.MachineLearning.Interop.csproj

```diff
@@ -2,7 +2,7 @@
 
   <PropertyGroup>
     <ProjectName>Microsoft.AI.MachineLearning.Interop</ProjectName>
-    <TargetFramework>netstandard2.0</TargetFramework>
```

Comment:
>>netstandard2. [](start = 20, length = 14)

We can not do it. Currently, the decision is to support netstandard2.0 for the widest possible user base.